app/src/main/res/values/component_colors.xml

```diff
@@ -46,4 +46,24 @@
   <color name="component_color_option_activity_text_color">@color/color_palette_dark_text_color</color>
   <color name="component_color_option_activity_options_background_color">@color/color_palette_container_background_color</color>
   <color name="component_color_option_activity_options_background_color_cyan">@color/color_palette_highlighted_background_color</color>
+  <!-- Hints And Solution Fragment -->
+  <color name="component_color_hints_and_solutions_fragment_background_color">@color/color_palette_secondary_background_color</color>
+  <color name="component_color_hints_and_solutions_fragment_text_color">@color/color_palette_highlighted_text_color</color>
+  <color name="component_color_hints_and_solutions_fragment_description_color">@color/color_palette_highlighted_text_color</color>
+  <color name="component_color_hints_and_solutions_divider_color">@color/color_palette_divider_color</color>
+  <!-- Story Activity -->
+  <color name="component_color_story_activity_background_color">@color/color_palette_primary_background_color</color>
+  <color name="component_color_story_activity_story_header_view_text_color">@color/color_palette_highlighted_text_color</color>
+  <color name="component_color_story_activity_container_background_color">@color/color_palette_primary_container_background_color</color>
+  <color name="component_color_story_activity_text_color">@color/color_palette_highlighted_text_color</color>
+  <!-- ResumeLessonActivity -->
```

Comment:
Write it as `Resume Lesson Activity `

Just to be consistent with your comments format. Overall file might not be consistent and that's fine for now.